fenix/app/onboarding.fml.yaml

```diff
@@ -11,7 +11,7 @@ features:
         default:
             default-browser:
               card-type: default-browser
-              title: juno_onboarding_default_browser_title_nimbus
+              title: juno_onboarding_default_browser_title_nimbus_2
               ordering: 10
               body: juno_onboarding_default_browser_description_nimbus
```

Comment:
We need to update all the other string IDs that you have updated here. Please make sure to also test this because you would've caught it yourself if you looked at it on a device.